code/pizzabot.js

```diff
@@ -1,7 +1,40 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const A = "Vegetarian Pizza"
```

Comment:
A better naming here would be nice! The names should explain what the data/value is. `A` doesn't say so much. It can be anything
```suggestion
const vegetarian = "Vegetarian Pizza"
```